ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,5 +4,14 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
+  m_Scenes:
+  - enabled: 1
+    path: Assets/Scenes/StartState.unity
+    guid: 17ba7c6ef1942584c8566f1b0cf6e57f
+  - enabled: 1
+    path: Assets/Scenes/GameStateTest.unity
+    guid: ee8b8a7d2168741a0a72adeaf09fb8c0
+  - enabled: 1
+    path: Assets/Scenes/FailState.unity
```

Comment:
It looks like this scene was deleted, it would be good to remove it from the build settings to prevent errors when we make a build